admin/meetings/views.py

```diff
@@ -0,0 +1,151 @@
+from __future__ import unicode_literals
+
+from django.views.generic import ListView, FormView
+from django.views.defaults import page_not_found
+from django.core.urlresolvers import reverse
+
+from framework.auth.core import get_user
+from website.models import Conference
+from website.conferences.exceptions import ConferenceError
+
+from admin.base.utils import OSFAdmin
+from admin.meetings.forms import MeetingForm
+from admin.meetings.serializers import serialize_meeting
+
+
+class MeetingListView(OSFAdmin, ListView):
+    template_name = 'meetings/list.html'
+    paginate_by = 10
+    paginate_orphans = 1
+    context_object_name = 'meeting'
+
+    def get_queryset(self):
+        return Conference.find()
+
+    def get_context_data(self, **kwargs):
+        queryset = kwargs.pop('object_list', self.object_list)
+        page_size = self.get_paginate_by(queryset)
+        paginator, page, queryset, is_paginated = self.paginate_queryset(
+            queryset, page_size
+        )
+        kwargs.setdefault('meetings', map(serialize_meeting, queryset))
+        kwargs.setdefault('page', page)
+        return super(MeetingListView, self).get_context_data(**kwargs)
+
+
+class MeetingFormView(OSFAdmin, FormView):
+    template_name = 'meetings/detail.html'
+    form_class = MeetingForm
+
+    def get(self, request, *args, **kwargs):
```

Comment:
These could probably go away and you could raise the correct `HttpXXX` exception with an error message.